packages/slave/slave.js

```diff
@@ -218,6 +233,7 @@ class Instance extends libLink.Link{
 	}
 
 	async init(pluginInfos) {
+		this.notifyStatus("starting");
```

Comment:
Maybe it's my state of being at 5am talking here, and I probably am overthinking this, but we could use something like `this.notifyStatus(new StartingMessage())`. But this s not PHP, loading classes is a manual job and in general I think I'm getting ahead of myself :D